packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -161,13 +161,15 @@ export class ChoiceGroup extends React.Component<IChoiceGroupProps, IChoiceGroup
   }
 
   private _onChange(option: IChoiceGroupOption, evt: React.FormEvent<HTMLInputElement>) {
-    let { onChanged } = this.props;
+    let { onChanged, onChange } = this.props;
 
     this.setState({
       keyChecked: option.key
     });
 
-    if (onChanged) {
+    if (onChange) {
+      onChange(evt, option);
+    } else if (onChanged) {
       onChanged(option);
```

Comment:
We actually do have a system for consistent console deprecation warnings. In BaseComponent, the constructor takes in props but 2nd param is a deprecation map. You can add an entry like:

"onChanged": "onChange"

And it will spit out a warning when onChanged is passed in. We use it in a bunch of places, but it's not well documented in the BESTPRACTICES.md. I will update it.